src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -449,6 +473,34 @@ public List<String> retrieveUserDatasetRoles(TestConfiguration.User user, UUID d
     return validateResponse(response, "retrieving dataset roles", HttpStatus.OK, null);
   }
 
+  // getting a resource's policies
+  public DataRepoResponse<PolicyResponse> retrievePoliciesRaw(
+      TestConfiguration.User user, UUID resourceId, IamResourceType iamResourceType)
+      throws Exception {
+    String pathPrefix;
+    switch (iamResourceType) {
+      case DATASET:
+        pathPrefix = "/api/repository/v1/datasets/";
+        break;
+      case DATASNAPSHOT:
+        pathPrefix = "/api/repository/v1/snapshots/";
+        break;
+      default:
+        throw new IllegalArgumentException(
+            "No path prefix defined for IamResourceType " + iamResourceType);
```

Comment:
Nit: could we write this with a format string? e.g.
```suggestion
            "No path prefix defined for IamResourceType {}", iamResourceType);
```

Reply:
Totally fair! I copy-pasted this from another method so I'll update them all.

Reply:
Using `{}` works in `log` but not in `Exception` constructors so I think you'd need to use `String.format()` or `formatted` here.

I've always wondered though, what's the motivation to avoid string concatenation in cases like this? Is using a format string easier to read? Personally I'd use concatenation in a case like this.

Reply:
After playing with this I agree with Phil here, I find the concatenated string easier to read and needing to explicitly format makes me wish to keep as is.